streams/tr.m3u

```diff
@@ -74,6 +82,10 @@ https://cdn-cekmekoybeltv.yayin.com.tr/cekmekoybeltv/cekmekoybeltv_1080p/playlis
 http://stream.taksimbilisim.com:1935/ciftcitv/smil:ciftcitv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CiftciTV.tr",Çiftçi TV (720p) [Not 24/7]
 https://live.artidijitalmedya.com/artidijital_ciftcitv/ciftcitv/playlist.m3u8
+#EXTINF:-1 tvg-id="CiftciTV.tr",Çiftçi TV (1080p) [Not 24/7]
+https://ythls.armelin.one/channel/UCtzcebnk0sLhCBtJy7GJaCg.m3u8
```

Comment:
Should be in streams/us_ythls.m3u.

Reply:
Why should an institution broadcasting in Turkish in Turkey be in us.m3u?